src/Arrays/matrix_operations.jl

```diff
@@ -240,3 +241,38 @@ function extend(M::AbstractMatrix; check_rank=true)
 
     return Mext, inv_Mext
 end
+
+"""
+    projection_matrix(block::AbstractVector{Int}, n::Int, [N]::DataType=Float64)
+
+Return the projection matrix associated to the given block of variables.
+
+### Input
+
+- `block` -- integer vector with the variables of interest
+- `n`     -- integer representing the ambient dimension
+- `N`     -- (optional, default: `Float64`) number type
+
+### Output
+
+
+A sparse matrix that corresponds to the projection onto the variables in `block`.
+
+### Examples
+
+```jldoctest projection_matrix
+julia> projection_matrix([1, 3], 4)
```

Comment:
Apparently we need `julia> LazySets.Arrays.projection_matrix([1, 3], 4)` here, or load `LazySets.Arrays` in these docs. @schillic  do you suggest something in particular? Otherwise i would just do `LazySets.Arrays.projection_matrix([1, 3], 4)

Reply:
I would do it as in a similar situation:

https://github.com/JuliaReach/LazySets.jl/blob/ef38a632e370dd745aa31dd32abd4bcc544d6c97/src/Arrays/vector_operations.jl#L79-L83

So:

````julia
```jldoctest projection_matrix
julia> using LazySets: projection_matrix

julia> projection_matrix([1, 3], 4)

[...]
````